ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcInputFormat.java

```diff
@@ -122,6 +122,7 @@ static class VectorizedOrcRecordReader
     public boolean next(NullWritable key, VectorizedRowBatch value) throws IOException {
 
       try {
+        rbCtx.resetVectorizedRowBatch(value, false);
```

Comment:
won't we have the same outcome if we call `addPartitionColsToBatch` regardless of `addPartitionCols` value?